src/metadata/metadata_status.h

```diff
@@ -200,14 +200,15 @@ static inline bool metadata_test_and_clear_dirty_sec(
 /*
  * Marks given cache line's bits as clean
  *
- * @return true if the cache line was dirty and became clean
+ * @return true if any cache line's sector was dirty and became clean
  * @return false for other cases
  */
 static inline bool metadata_clear_dirty_sec_changed(
 		struct ocf_cache *cache, ocf_cache_line_t line,
-		uint8_t start, uint8_t stop)
+		uint8_t start, uint8_t stop, bool *became_clean)
```

Comment:
I suggest to revert the logic of last parameter and rename it to "line_was_clean", similarly to what metadata_set_dirty_sec_changed's  'line_was_dirty'.

This way the two functions are used similarily:
if (metadata_(clear/set)_dirty_sec_changed(..., &line_was(clear/dirty)) {
    mark_flush()
}
if (!line_was(clear/dirty)) {
   //udpate stats
}

Reply:
Information about previous state of cache line is equivalent with information retrieved from metadata_clear_dirty_sec_changed() return value. Information that we need to have is new status of whole cache line  - if it became clean or not. And only if it did, we can change cache stats.